app/src/main/java/ar/com/wolox/android/training/BaseConfiguration.kt

```diff
@@ -3,7 +3,7 @@ package ar.com.wolox.android.training
 internal open class BaseConfiguration {
 
     companion object {
-        const val TRAINING_CONFIGURATION_KEY = "https://jsonplaceholder.typicode.com"
+        const val TRAINING_CONFIGURATION_KEY = "https://android-training.herokuapp.com"
```

Comment:
You can rename this const to API_URL or BASE_API or something easier to find afterwards